common/components/internal-header/_internal-header.scss

```diff
@@ -2,310 +2,179 @@ $govuk-header-background: govuk-colour("black");
 
-  .app-header__menu-button {
-    right: govuk-spacing(3);
+  @media print {
+    display: none;
   }
 }
 
-.app-header__container--with-border {
-  margin-bottom: -$govuk-header-border-width;
-  border-bottom: $govuk-header-border-width solid $govuk-header-border-color;
+.app-header__content {
+  flex-direction: row;
+  display: flex;
+  align-self: stretch;
 }
 
 .app-header__container {
-  @include govuk-clearfix;
-  position: relative;
-  padding-top: govuk-spacing(2);
-}
+  max-width: $govuk-page-width;
+  margin-right: govuk-spacing(3);
+  margin-left: govuk-spacing(3);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @media (min-width: 40.0625em) {
+    margin-right: govuk-spacing(6);
+    margin-left: govuk-spacing(6);
+  }
+
+  @supports (margin: Max(Calc(0px))) {
+    margin-right: Max(
+      govuk-spacing(3),
+      Calc(govuk-spacing(3) + #{env(safe-area-inset-right)})
+    );
+    margin-left: Max(
+      govuk-spacing(3),
+      Calc(govuk-spacing(3) + #{env(safe-area-inset-left)})
+    );
+
+    @media (min-width: 40.0625em) {
+      margin-right: Max(
+        govuk-spacing(6),
+        Calc(govuk-spacing(3) + #{env(safe-area-inset-right)})
+      );
+      margin-left: Max(
+        govuk-spacing(6),
+        Calc(govuk-spacing(3) + #{env(safe-area-inset-left)})
+      );
+    }
+  }
 
-.app-header__logotype {
-  margin-right: govuk-spacing(1);
+  @media (min-width: 1230px) {
+    margin-right: auto;
+    margin-left: auto;
+  }
 }
 
 .app-header__logotype-crest {
-  margin-right: 1px;
-  fill: currentColor;
-  vertical-align: middle;
+  margin-right: govuk-spacing(2);
+  position: relative;
+  top: -2px;
+  fill: $govuk-header-crest;
 }
 
-.app-header__logotype-crest-fallback-image {
-  width: 42px;
-  height: 35px;
-  border: 0;
-  vertical-align: middle;
+.app-header__logo {
+  padding-right: govuk-spacing(3);
+  display: flex;
+  align-items: center;
 }
 
-.app-header__product-name {
-  @include govuk-font($size: 24);
-  display: inline-table;
+.app-header__logotype {
+  margin-right: govuk-spacing(2);
+  @include govuk-font($size: 24, $weight: bold);
+  display: flex;
+  align-items: center;
 }
 
 .app-header__link {
-  text-decoration: none;
+  @include govuk-font($size: 19);
 
   &:link,
-  &:visited {
+  &:visited,
+  &:active {
     color: $govuk-header-link;
+    text-decoration: none;
   }
 
   &:hover {
     text-decoration: underline;
   }
 
-  // When focussed, the text colour needs to be darker to ensure that colour
-  // contrast is still acceptable
   &:focus {
     @include govuk-focused-text;
-  }
 
-  // alphagov/govuk_template includes a specific a:link:focus selector
-  // designed to make unvisited links a slightly darker blue when focussed, so
-  // we need to override the text colour for that combination of selectors.
-  @include govuk-compatibility(govuk_template) {
-    &:link:focus {
-      @include govuk-text-colour;
+    svg {
```

Comment:
This looks like its nested within the `@include govuk-compatibility`, is that GitHub diffs playing up or intentional?

I'm guessing we might not need it nested if so?

Reply:
That's just GitHub diffs playing up, this is nested as `.app-header__link { &:focus { svg {`